### PR TITLE
[webOS] Acb: Install empty dummy library

### LINKED ISSUE #23897: [webOS] Crash when playing AV1 video file
### Describe the bug
Cannot play an AV1 file.

## Expected Behavior
The file should play.

## Actual Behavior
Kodi crashes.

## Possible Fix
Downgrading to 35680593 fixes the issue.

### To Reproduce
1. Launch Kodi on webOS 5+ with AV1 support
2. Try playing an AV1 file
3. Kodi crashes

### Debuglog
The debuglog can be found here: [kodi.log](https://github.com/xbmc/xbmc/files/12850986/kodi.log)
Crash report: [2023-09-04-nightly-crash-report.txt](https://github.com/xbmc/xbmc/files/12850926/2023-09-04-nightly-crash-report.txt)

## Additional context or screenshots (if appropriate)
This seems to be a regression caused by https://github.com/xbmc/xbmc/pull/23834

### Your Environment
Used Operating system:
 - [x] webOS

 - Operating system version/name: webOS 7, LG OLED C2
 - Kodi version: `org.xbmc.kodi_20231004-0f289470-master_arm.ipk`

### PR DESCRIPTION
**Pull request by sundermann**
## Description

For compability reasons we need to ship the `libAcbAPI` library for webOS 5+ devices. These used to be shipped from webos-userland. These libraries are merely used as a linking target and contain no code at all. They are generated using some linker script tricks. For some strange reason having these libraries present on webOS 5+ results in AV1 files not playing as reported in #23897. Functions from `libAcbAPI` are never called on webOS 5+. It is possible that the linker script tricks end up with a corrupted library but the exact reason for why the library segfaults for AV1 files is unknown. As a workaround compile a empty library and install that instead.

## Motivation and context

Fix #23897

## How has this been tested?

webOS 7/22

## Types of change

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **Student submission** (PR was done for educational purposes and will be treated as such)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [x] All new and existing tests passed

### CODE REVIEW

File: cmake/scripts/webos/Install.cmake [hunk 1]
```diff
@@ -67,6 +67,10 @@ add_custom_command(
   VERBATIM
 )
 
+file(TOUCH dummy.c)
+add_library(AcbAPI SHARED dummy.c)
+set_target_properties(AcbAPI PROPERTIES VERSION 1.0.0 SOVERSION 1)
```
Comment (wsnipex) [inline cmake/scripts/webos/Install.cmake:72]: I'd move this to the find module, for the lack of a better place

Comment (sundermann) [reply cmake/scripts/webos/Install.cmake]: Moved it over

**Review by wsnipex — APPROVED** [review]

**Comment by sundermann** [comment]
@quietvoid Can you give this a try to see if it also fixes AV1 playback for you?

**Comment by quietvoid** [comment]
Hi, I tested the build from https://jenkins.kodi.tv/job/webos-docker/1992/ and it works fine with AV1 files.
Thanks!

**Pull request merged**